### ISSUE TITLE
Term Search Dropdown - 3.3.2 Labels or Instructions - Three missing form labels are present

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9822]
As a user reliant on the screen reader for discerning the purpose of form fields, I expect the screen reader to be able to convey the name, purpose and instructions for use of every form field, facilitated by form labels in the HTML.

**Linked work item:** 
#9499 

**Expected Result:**
No missing form label errors should be present. Form labels should be present on all form fields to help the screen reader interpret the purpose of a field and instructions for its use.

**Actual Result:**
Three missing form labels on the search dropdown component; one above the search bar, one on the search bar itself and one further on the language search dropdown.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/901aeae9-587f-4c3c-a3af-b8b087ac0b63)

**Comment by johnatawnclementawn** [comment]
This issue is dependent on our select2/chosen replacement work

**Comment by apeters** [comment]
@imogenroberts this should be fixed in the latest

**Comment by imogenroberts** [comment]
RETEST: No missing form labels are present on this or any other component in the page. 

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/23e342e8-c045-44a8-bf0b-0f0d20c9a48b)
